lib/util/SortableSet.js

```diff
@@ -1,17 +1,30 @@
 "use strict";
 
+/**
+ * A subset of Set that offers sorting functionality
```

Comment:
`subset` is the wrong word here. Maybe `superset`?

Reply:
You are right. Superset is the right word.

Reply:
Why not subtype or subclass?

Reply:
it adds functionality to Set. A superset of Set and a subclass of Set at the same time. I don't know, my TypeScript is probably better than my English.